src/actions/__tests__/range-test.js

```diff
@@ -3,7 +3,7 @@ import getRange from '../range'
 describe('actions/getRange', () => {
   it('returns range between 2 and totalPage if totalPage' +
```

Comment:
preset range is the totalNumButton i.e how many number you want to show in the range.
<img width="652" alt="range" src="https://cloud.githubusercontent.com/assets/21019282/25662553/8931c3f8-3047-11e7-8eb9-2a4811920560.png">